demisto_sdk/common/hook_validations/integration.py

```diff
@@ -471,3 +473,10 @@ def is_not_valid_display_configuration(self):
                 return True
 
         return False
+
+    def is_docker_image_valid(self):
+        # type: () -> bool
+        docker_image_validator = DockerImageValidator(self.file_path, is_modified_file=True, is_integration=True)
+        if not docker_image_validator.is_docker_image_valid():
+            return True
```

Comment:
please update the `self.is_valid = False` here and after the if add the self.is_valid = True please